packages/core/database/manager.go

```diff
@@ -261,14 +258,14 @@ func (m *Manager) PrunableStorageSize() int64 {
 	})
 
 	// Add up all the open databases
-	m.openDBs.Each(func(key epoch.Index, val *dbInstance) {
-		size, err := dbPrunableDirectorySize(m.bucketedBaseDir, key)
-		if err != nil {
-			fmt.Println("dbPrunableDirectorySize failed for", m.bucketedBaseDir, key, ":", err)
-			return
-		}
-		sum += size
-	})
+	//m.openDBs.Each(func(key epoch.Index, val *dbInstance) {
```

Comment:
This code is needed to calculate the size of the current open DBs. Yes, there could be errors while traversing the filesystem, because RocksDB might have compacted files while we traverse